src/pages/executeFlow/Execute.tsx

```diff
@@ -130,273 +156,306 @@ function ExecuteFlow({
-              <tds-textarea
-                rows="20"
-                disabled={!enableEditParameter}
-                label="JSON"
-                label-position="outside"
-                placeholder="Placeholder"
-                value={selectedParameter.value}
-              ></tds-textarea>
+            <div className="tds-detail-02">
+              IRI: <span className="bold">{iri}</span>
             </div>
-          )}
-
-          <div className={styles.action}>
-            {mode === "newParameter" && (
-              <>
-                <tds-button
-                  type="button"
-                  size="sm"
-                  variant="secondary"
-                  text="Back"
-                  onClick={() => changeMode("initial")}
-                ></tds-button>
-                <tds-button
-                  type="button"
-                  size="sm"
-                  variant="primary"
-                  text="Save parameter"
-                  onClick={() => saveParameter()}
-                ></tds-button>
-                {/*
+          </div>
+        </div>
+        <div className={styles.tabs}>
+          <Tabs
+            selectedIndex={selectedTab}
+            onParentClick={(value: string) => setSelectedTab(value)}
+          >
+            {[<Panel title="Execution" value="Execution"></Panel>]}
+            {/* Result tab hidden untill functionality added
+              <Panel title="Results" value="Results"></Panel>
+            */}
+          </Tabs>
+          <div className={styles.outerContentContainer}>
+            <div className={styles.contentContainer}>
+              {selectedTab === "Execution" && (
+                <div>
+                  <h6 className="tds-headline-06">Execute parameter</h6>
+                  <hr className="divider" />
+                  <div className={styles.contentContainer__parameterChoice}>
+                    <tds-radio-button
+                      name="rb-example"
+                      value="newParam"
+                      radio-id="create"
+                      onClick={(e) => {
+                        setSelectedExecutionMethod("Create");
+                        setSelectedParameter(null);
+                      }}
+                      checked={selectedExecutionMethod === "Create"}
+                    >
+                      <div slot="label">Create new parameter</div>
+                    </tds-radio-button>
 
-                TODO - executing a graph with a newly created parameter might be a bit tricky, holding on this one for now
+                    <tds-radio-button
+                      name="rb-example"
+                      onClick={() => {
+                        parameters.length &&
+                          setSelectedExecutionMethod("Existing");
+                      }}
+                      disabled={!parameters.length}
+                      value="savedParams"
+                      radio-id="execute"
+                      checked={
```

Comment:
The checked value is for initial render I believe, we should be able to skip these and just go with "checked" on the option that we want to have pre-selected